### ISSUE TITLE
Not getting MMS notifications when SMS notifications are enabled

### ISSUE THREAD

**Issue by glitchphan** [Freeyourgadget/Gadgetbridge#127]
This may or may not be a Gadgetbridge thing, but I never get any notification on the watch when I get an MMS (which includes multi-recipient texts from iPhones). I just had a friend send me a picture message while my screen was on, and the notification from Qksms looks the same as the regular SMS one.

**Comment by ashimokawa** [comment]
After the next release you could set SMS notificaion to "never", then SMS will be handled as a generic notification. Maybe that works. Did the MMS contain zero text and only the Photo?

**Comment by glitchphan** [comment]
Just got to do some MMS testing. With SMS notifications set to "when screen is off", neither the MMS with text nor without text came through to the watch. When I set SMS to "never", nothing came through with just a picture, but the text did show up when it was along with a picture. Regular SMS comes through as well, and the icons/color (purple) are no different with both types of messages and both settings. I did not try receiving a multi-recipient message yet. This is all with QKSms. Hope it made sense :)
